manager/volume.go

```diff
@@ -35,6 +63,44 @@ func (m *VolumeManager) GetCurrentNodeID() string {
 	return m.currentNodeID
 }
 
+func (m *VolumeManager) GetSupportBundleSize() int64 {
+	return m.sb.Bsize
+}
+
+func (m *VolumeManager) GetSupportBundleState() BundleState {
+	return m.sb.state
+}
+
+func (m *VolumeManager) SBDownloadInProgress() bool {
+	return m.GetSupportBundleState() == InProgress || m.GetSupportBundleState() == Ready
```

Comment:
This looks wrong... InProgress is not InProgress or Ready...

Also, either always use the full name `Support Bundle` or short name `SB`. I would prefer the full name.